src/QueryBuilder/QueryBuilder/helpers/valueBuilder.js

```diff
@@ -27,6 +27,10 @@ export const getFormattedUUID = (value, isInRelatedOperator) => {
 };
 
 const formatDateToPreview = (dateString, intl, timezone) => {
+  if (typeof dateString === 'boolean') {
+    return dateString;
+  }
+
```

Comment:
Would a better fix be to not show the `Value` field at all when the `Operator` field is set to `is null/empty`? What is displayed when somebody sets the `Value` field to `True`?

It seems [the `valueBuilder` test for this condition](https://github.com/folio-org/ui-plugin-query-builder/blob/52bb1fcab2ed2a25fef5b18b985a954bfc321916/src/QueryBuilder/QueryBuilder/helpers/valueBuilder.test.js#L62-L68) is inadequate since the type is `boolean` but the value is `false`, a (non-falsy) string. Please update this test.

Reply:
<img width="1094" alt="image" src="https://github.com/user-attachments/assets/29626897-13ec-47e8-bccd-6b2c296b3c9a" />
it's required only for UI side to fix how dates are displaying, coz we have an operator that is `null/empty`. we can see the problem if we select `null/empty` and use const `formattedDate = dayjs.utc(dateString)`; with `true/false` we see the first possible date from JS.

Reply:
I don't entirely understand this but I'm going to take your word for it since it's clear you understand all of this better than I do.